<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="6" failed="0" ignored="16" total="22" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Payment Gateway Project Test Automation QA Staging Domainz Test Suite" duration-ms="330" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
    <groups>
    </groups>
    <test name="Payment Gateway Test Cases" duration-ms="330" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
      <class name="com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest">
        <test-method status="FAIL" signature="setUp(java.lang.String)[pri:0, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="setUp" is-config="true" duration-ms="24" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[stagingdomainz]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )

at com.base.TestBase.initialization(TestBase.java:36)
at com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest.setUp(DomainzCustomerCartJourneyTest.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="verifyDomainRegistrationForNewCustomer()[pri:1, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="verifyDomainRegistrationForNewCustomer" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )

at com.base.TestBase.initialization(TestBase.java:36)
at com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest.setUp(DomainzCustomerCartJourneyTest.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyDomainRegistrationForNewCustomer -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="setUp(java.lang.String)[pri:0, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="setUp" is-config="true" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="verifyDomainRegistrationForExistingCustomer()[pri:2, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="verifyDomainRegistrationForExistingCustomer" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )

at com.base.TestBase.initialization(TestBase.java:36)
at com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest.setUp(DomainzCustomerCartJourneyTest.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyDomainRegistrationForExistingCustomer -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="setUp(java.lang.String)[pri:0, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="setUp" is-config="true" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="verifyHostingSignupForNewCustomer()[pri:3, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="verifyHostingSignupForNewCustomer" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )

at com.base.TestBase.initialization(TestBase.java:36)
at com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest.setUp(DomainzCustomerCartJourneyTest.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHostingSignupForNewCustomer -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="setUp(java.lang.String)[pri:0, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="setUp" is-config="true" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="verifyHostingSignupForExistingCustomer()[pri:4, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="verifyHostingSignupForExistingCustomer" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )

at com.base.TestBase.initialization(TestBase.java:36)
at com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest.setUp(DomainzCustomerCartJourneyTest.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHostingSignupForExistingCustomer -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="setUp(java.lang.String)[pri:0, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="setUp" is-config="true" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="verifySEOForNewCustomer()[pri:13, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="verifySEOForNewCustomer" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )

at com.base.TestBase.initialization(TestBase.java:36)
at com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest.setUp(DomainzCustomerCartJourneyTest.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySEOForNewCustomer -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="setUp(java.lang.String)[pri:0, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="setUp" is-config="true" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="verifySEOForExistingCustomer()[pri:14, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="verifySEOForExistingCustomer" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )

at com.base.TestBase.initialization(TestBase.java:36)
at com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest.setUp(DomainzCustomerCartJourneyTest.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySEOForExistingCustomer -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest@146ba0ac]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-05-07T23:49:23Z" finished-at="2018-05-07T23:49:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.paymentgateway.ui.dmz.testcases.DomainzCustomerCartJourneyTest -->
    </test> <!-- Payment Gateway Test Cases -->
  </suite> <!-- Payment Gateway Project Test Automation QA Staging Domainz Test Suite -->
</testng-results>
